### PR TITLE
{bio}[foss/2022a] GROMACS v2023.1 w/wo CUDA 11.7.0

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
b-cn1603.hpc2n.umu.se - Linux Ubuntu 22.04, x86_64, AMD EPYC 7313 16-Core Processor, 1 x NVIDIA NVIDIA A100 80GB PCIe, 525.116.04, Python 3.10.6
See https://gist.github.com/akesandgren/f1bed348a43e0626bd2a40d3aca900a7 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
b-cn1603.hpc2n.umu.se - Linux Ubuntu 22.04, x86_64, AMD EPYC 7313 16-Core Processor, 1 x NVIDIA NVIDIA A100 80GB PCIe, 525.116.04, Python 3.10.6
See https://gist.github.com/akesandgren/f4ebc8496239e827b666c5281109b2fe for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0103u01a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6330 CPU @ 2.00GHz (icelake), 1 x NVIDIA NVIDIA A100-PCIE-40GB, 520.61.05, Python 3.6.8
See https://gist.github.com/branfosj/df8fcf5a0d4b4df826236ae207a0ae9d for a full test report.

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2023.1-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'use_pip': True,
+    'download_dep_fail': True,
+    'sanity_pip_check': True,
+}
+
+exts_list = [
+    ('gmxapi', '0.4.0', {
+        'preinstallopts': 'export CMAKE_ARGS="-Dgmxapi_ROOT=%(installdir)s -C %(installdir)s/share/cmake/gromacs_mpi/gromacs-hints_mpi.cmake" && ',
```
Comment (branfosj) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2023.1-foss-2022a.eb:77]:
```suggestion
        'preinstallopts': 'export CMAKE_ARGS="-Dgmxapi_ROOT=%(installdir)s ' +
                          '-C %(installdir)s/share/cmake/gromacs_mpi/gromacs-hints_mpi.cmake" && ',
```

Re-split long line that got unsplit by injecting checksums.

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2023.1-foss-2022a-CUDA-11.7.0.eb [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'use_pip': True,
+    'download_dep_fail': True,
+    'sanity_pip_check': True,
+}
+
+exts_list = [
+    ('gmxapi', '0.4.0', {
+        'preinstallopts': 'export CMAKE_ARGS="-Dgmxapi_ROOT=%(installdir)s -C %(installdir)s/share/cmake/gromacs_mpi/gromacs-hints_mpi.cmake" && ',
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/g/GROMACS/GROMACS-2023.1-foss-2022a-CUDA-11.7.0.eb:79]: Gah didn't check for that...

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0203u29a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), 1 x NVIDIA NVIDIA A100-SXM4-80GB, 520.61.05, Python 3.6.8
See https://gist.github.com/branfosj/d83c7a1166d4d421aa7a9677d86ea654 for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
Going in, thanks @akesandgren!

**Pull request merged**